### PR TITLE
Added an option to print out "messages" while exporting a .trace file to CSV using "csvexport" utility

### PR DESCRIPTION
**Pull request by nehasharan**

### CODE REVIEW

File: csvexport/src/csvexport.cpp [hunk 1]
```diff
@@ -23,12 +23,13 @@ void print_usage_exit(int e)
...
-    fprintf(stderr, "  -c, --case        Case sensitive filtering\n");
-    fprintf(stderr, "  -e, --self        Get self times\n");
-    fprintf(stderr, "  -u, --unwrap      Report each zone event\n");
+    fprintf(stderr, "  -h, --help                       Print usage\n");
+    fprintf(stderr, "  -f, --filter arg                 Filter zone names (default: "")\n");
+    fprintf(stderr, "  -s, --sep arg                    CSV separator (default: ,)\n");
+    fprintf(stderr, "  -c, --case                       Case sensitive filtering\n");
+    fprintf(stderr, "  -e, --self                       Get self times\n");
+    fprintf(stderr, "  -u, --unwrap                     Report each zone event\n");
+    fprintf(stderr, "  -m, --unwrap only messages       Report only messages\n");
```
Comment (wolfpld) [inline csvexport/src/csvexport.cpp:32]: This does not correspond with what the program expects.

Comment (nehasharan) [reply csvexport/src/csvexport.cpp:32]: Hi @wolfpld there is currently no way to print messages which we add to zones.

Comment (fosterbrereton) [reply csvexport/src/csvexport.cpp:32]: @wolfpld would this be more acceptable?
```suggestion
    fprintf(stderr, "  -m, --messages                   Report only messages\n");
```

Comment (wolfpld) [reply csvexport/src/csvexport.cpp:32]: Yes, but the additional padding added between arguments and their descriptions is no longer needed.

File: csvexport/src/csvexport.cpp [hunk 2]
```diff
@@ -198,6 +204,38 @@ int main(int argc, char** argv)
...
+                std::string row = join(values, args.separator);
+                printf("%s\n", row.data());
+            }
+        }
+        else
+        {
+            printf("There are currently no messages!\n");
+        }
+    
+    return 0;
```
Comment (wolfpld) [inline csvexport/src/csvexport.cpp:236]: Align return with indentation.

Comment (nehasharan) [reply csvexport/src/csvexport.cpp]: @wolfpld done.

**Pull request merged**